src/react-components/home/HomePage.scss

```diff
@@ -35,8 +35,16 @@
 
 :local(.sign-in-container) {
```

Comment:
I was confused about re-using this classname, but I don't actually see where it was being used before...

Reply:
I think it was some unused styling before.